content/intermediate/templates/nested-stacks/index.md

```diff
@@ -468,7 +576,7 @@ If you not sure how to do that, follow the instructions from the [Session Manage
 
 Follow these steps to clean up created resources:
 
-1. In the **[CloudFormation console](https://console.aws.amazon.com/cloudformation)**, select the **root** stack you have created in this lab. For example `cfn-workshop-nested-stack`.
+1. In the **[CloudFormation console](https://console.aws.amazon.com/cloudformation)**, select the **root** stack you have created in this lab. For example `cfn-workshop-nested-stacks`.
 1. The **root** stack will handle the deletion of all the **children** stacks for you.
 1. In the top right corner, click on **Delete**.
 1. In the pop-up window click on **Delete stack**.
```

Comment:
```suggestion
1. In the pop-up window click on **Delete**.
```

Should we also suggest emptying/deleting the S3 bucket used for templates?

Reply:
That bucket was created in some other lab, not sure deleting that now will impact something later. so safer side for now we can skip that.